osu.Game/Screens/Select/FooterButton.cs

```diff
@@ -6,183 +6,234 @@
 using System;
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Effects;
 using osu.Framework.Graphics.Shapes;
 using osu.Framework.Graphics.Sprites;
 using osu.Framework.Input.Bindings;
 using osu.Framework.Input.Events;
 using osu.Framework.Localisation;
+using osu.Game.Graphics;
 using osu.Game.Graphics.Containers;
 using osu.Game.Graphics.Sprites;
 using osu.Game.Input.Bindings;
 using osuTK;
-using osuTK.Graphics;
 
 namespace osu.Game.Screens.Select
 {
-    public class FooterButton : OsuClickableContainer, IKeyBindingHandler<GlobalAction>
+    public abstract class FooterButton : OsuClickableContainer, IKeyBindingHandler<GlobalAction>
     {
-        public const float SHEAR_WIDTH = 7.5f;
+        public const float SHEAR_WIDTH = 16;
+        private const float outer_corner_radius = 10;
+        private const int ease_out_time = 800;
+        private const int button_height = 120;
+        private const float shear_padding = 10;
+        protected readonly FontUsage TorusFont = OsuFont.TorusAlternate.With(size: 16);
 
-        protected static readonly Vector2 SHEAR = new Vector2(SHEAR_WIDTH / Footer.HEIGHT, 0);
+        private readonly Colour4 backgroundColour = Colour4.FromHex("#394642");
```

Comment:
Where did this colour come from?

Reply:
As with all the others I sourced them straight from figma.

Reply:
And this colour doesn't match an existing scheme? Can you link or screenshot where you're seeing this?

Reply:
sorry i should've looked for matching schemes, will address, but still linking this as proof i didn't magic the colour out of nowhere,
![sourced colour](https://user-images.githubusercontent.com/74463310/188596929-9e2493aa-a49a-4efb-841c-0e57c10a59cb.png)

Reply:
I'm fairly certain the `B/B3` in the `Colours` section of figma indicates that this is a [Background3](https://github.com/ppy/osu/blob/6296c971f8d7a8820b3f40162d33b7d219b7d0c9/osu.Game/Overlays/OverlayColourProvider.cs#L47) which is pretty much the same as the `Aquamarine` scheme (#394641 vs #394642)